string[] obj_name
string[] obj_frame

float32[] obj_angle_horizontal
float32[] obj_angle_vertical

int32[] obj_pixel_x
int32[] obj_pixel_y

int32[] obj_pixel_left_x
int32[] obj_pixel_top_y

int32[] obj_pixel_width
int32[] obj_pixel_height

float32[] obj_meter_x
float32[] obj_meter_y
float32[] obj_meter_z

float32[] obj_meter_depth